.github/workflows/docker.yaml

```
@@ -21,17 +24,22 @@ jobs:
 
       - name: Build image
         run: docker build -t image .
+        
+      - name: Sanitize repo slug
+        uses: actions/github-script@v4
```

I think we should use the latest version of this action

```suggestion
        uses: actions/github-script@v6
```